src/server/scripts/Northrend/Ulduar/HallsOfStone/boss_sjonnir.cpp

```diff
@@ -155,7 +153,7 @@ class boss_sjonnir : public CreatureScript
                     {
                         brann->setDeathState(DeathState::JustDied);
                         brann->Respawn();
-                        brann->AI()->DoAction(5);
+                        brann->AI()->DoAction(7);   //ACTION_SJONNIR_WIPE_START
```

Comment:
please remote the magic number here, would probably require moving the relevant brann defines into a header file that both script cpps can include.